app/helpers/application_helper.rb

```diff
@@ -345,11 +345,18 @@ def email_as_id(email)
   
 	# Return an access token for viewing GCS objects client side, depending on study privacy
 	# Context: https://github.com/broadinstitute/single_cell_portal_core/pull/239
-	def get_read_access_token(study, user)
+  def get_read_access_token(study, user)
     if study.public? && Study.read_only_firecloud_client.present?
       Study.read_only_firecloud_client.valid_access_token["access_token"]
     elsif user.present?
       user.valid_access_token[:access_token]
+    end
+  end
+
+  # Return the user's access token for bulk download of faceted search results
+  def get_user_access_token(user)
+    if user.present?
+      user.valid_access_token[:access_token]
     end
 	end
```

Comment:
It looks like there's some formatting issues here w/ newlines

Reply:
Fixed in cc00043.  Oddly, VS Code doesn't show this formatting issue.